MvvmCross/Droid/Shared/Presenter/MvxFragmentsPresenter.cs

```diff
@@ -21,8 +21,8 @@ public class MvxFragmentsPresenter
     {
         public const string ViewModelRequestBundleKey = "__mvxViewModelRequest";
 
-        private readonly FragmentHostRegistrationSettings _fragmentHostRegistrationSettings;
-        private readonly Lazy<IMvxNavigationSerializer> _lazyNavigationSerializerFactory;
+        protected FragmentHostRegistrationSettings _fragmentHostRegistrationSettings;
+        protected Lazy<IMvxNavigationSerializer> _lazyNavigationSerializerFactory;
```

Comment:
Well, i made some up the methods in `FragmentHostRegistrationSettings` virtual so people can make their own class and change that behaviour. They would need to set this some way.